docs/lib/DEBUGGING.md

```diff
@@ -144,6 +144,18 @@ The OpenSDA port provides a debugger interface, which is compatible with the Seg
 
 To use the JTAG port instead, download the tools and drivers for the debugger of your choice.
 
+#### ULINK pro configuration for NXP FRDM-K64F
+
+Configure NXP FRDM-K64F according to the pdf document from arm Keil [Application Note](http://www.keil.com/appnotes/docs/apnt_287.asp).
+
+In order to add an example to uvision run the "mbed export -i UVISION5 -m K64F" command, and open the project from uvision with the .uvprojx file created.
```

Comment:
please consider adding a link to official mbed-os tutorial
https://developer.mbed.org/handbook/Exporting-to-uVision